### PR TITLE
Add SCP workaround

### PR DESCRIPTION
**Pull request by jayson-jensen-pro**

### CODE REVIEW

File: faqs/faqs-aws.md [hunk 1]
```diff
@@ -35,6 +35,44 @@ To skip certain sub-accounts when auto-configuring JupiterOne AWS integrations
...
+  "Statement": {
+    "Sid": "...",
+    "Effect": "...",
+    "Action": "...",
+    "NotAction": "...",
+    "Resource": "...",
+    "Condition": {
+      "ArnNotLike": {
+        "aws:PrincipalARN": [
+            "arn:aws:iam::*:role/JupiterOne*",
```
Comment (erkangz) [inline faqs/faqs-aws.md:68]: we should include the J1 account ID here, otherwise this opens it up to _any_ account with a role named `JupiterOne*`

Comment (a-u-h-g) [reply faqs/faqs-aws.md:68]: The ARN would contain the customer's AWS account ID, not ours. Without an external account trust on the IAM Role, an IAM Role from another account should not be able to access their account.

File: faqs/faqs-aws.md [hunk 2]
```diff
@@ -35,6 +35,34 @@ To skip certain sub-accounts when auto-configuring JupiterOne AWS integrations
...
+element to your SCP JSON: 
+
+**Note** Make sure the ARN below matches the IAM Role ARN you used to configure
+your JupiterOne AWS integration.
+
+```json
+"Condition": {
+  "ArnNotLike": {
+    "aws:PrincipalARN": [
+        "arn:aws:iam::*:role/JupiterOne*",
```
Comment (a-u-h-g) [inline faqs/faqs-aws.md:60]:
```suggestion
        "arn:aws:iam::*:role/JupiterOne*"
```

**Review by a-u-h-g — APPROVED** [review]

**Review by a-u-h-g — CHANGES_REQUESTED** [review]
extra comma in JSON snippet

**Review by a-u-h-g — APPROVED** [review]

**Review by erkangz — APPROVED** [review]

File: faqs/faqs-aws.md (hunk 1 above)
Comment (erkangz) [reply faqs/faqs-aws.md]: You are right. I misread it.

**Pull request merged**